### PR TITLE
fixes options for ansible pull

### LINKED ISSUE #12955: ansible-pull 2.0.0 requires an argument
with ansible 2.0.0-beta2 I get:

```
# ansible-pull --url myurl --key-file key
Usage: ansible-pull <host-pattern> [options]
....
ERROR! Missing target hosts
```

Man page still indicates that it should default to local.yml.  Seems something is messed up here.

### PR DESCRIPTION
**Pull request by bcoca**
- remove requirement for host patterns, use the defaults
- require destination directory (None in cwd is not a good default)
- fixed usage messages
- updated default inventory to use , and not deprecated

fixes #12955

### CODE REVIEW

**Comment by abadger** [comment]
Code looks fine.  #12955 also asks for man page update.  And we should document in chaneglog if the CLI is incompatible (requires dest) as well.

**Comment by bcoca** [comment]
man page is already updated

**Comment by abadger** [comment]
```
*-d* 'DEST', *--directory=*'DEST'::

Directory  to checkout repository into. If not provided, a subdi-
rectory of ~/.ansible/pull/ will be used.
```

That's no longer true, correct?

File: lib/ansible/cli/pull.py [hunk 1]
```diff
@@ -129,6 +126,7 @@ def run(self):
         #TODO: enable more repo modules hg/svn?
         if self.options.module_name == 'git':
             repo_opts = "name=%s dest=%s" % (self.options.url, self.options.dest)
+            print(repo_opts)
```
Comment (abadger) [inline lib/ansible/cli/pull.py]: Is this debugging that should be removed?

**Comment by bcoca** [comment]
#totallymeanttodothat

**Comment by jimi-c** [comment]
Merged, thanks!

**Pull request merged**